rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -25,24 +25,20 @@ class QuotingPanel {
 
     let totalAreaInMts = 0;
     this.polygons = polygons;
-    console.log("this.polygons: ", this.polygons)
     polygons.forEach((p) => { 
       let areaInMts = google.maps.geometry.spherical.computeArea(p.getPath());
-      console.log("Area (mts): ", areaInMts)
       totalAreaInMts += areaInMts;
-      console.log("Total area(mts): ", totalAreaInMts)
     });
 
     this.totalAreaInSqFt = totalAreaInMts * SQ_FT_CONVERT;
-    console.log("Total area(FT): ", this.totalAreaInSqFt)
 
     let areaNode = document.getElementById("calculatedArea");
-    areaNode.innerText = `Selected area: ${this.totalAreaInSqFt.toFixed(0)} sq ft`;
-    areaNode.parentNode.classList.remove("hidden");
+    areaNode.innerText = `${this.totalAreaInSqFt.toFixed(0)}`;
+    areaNode.parentNode.parentNode.classList.remove("hidden");
 
     this.totalDue = this.totalAreaInSqFt * PRICE_PER_SQ_FT;
     let totalDueNode = document.getElementById("totalDue");
-    totalDueNode.innerText = `Total    $${this.totalDue.toFixed(2)}`;
+    totalDueNode.innerText = `${this.totalDue.toFixed(2)}`;
```

Comment:
https://stackoverflow.com/a/2901298/302246

Reply:
I have added a ticket to fix this on my quotes_panel branch